### ISSUE TITLE
Ensure all reports pages load with date range by default

### ISSUE THREAD

**Issue by Matt-Yorkley** [openfoodfoundation/openfoodnetwork#5032]
We should make sure _all_ reports pages load with date range set in the forms by default. Currently there's some that don't...

**Comment by luchiago** [comment]
Hi, I would like to work on this issue. Can you detail more the problem?

**Comment by RachL** [comment]
@Matt-Yorkley just a feedback: since we put the date range by default on the OC reports, I get only bad feedback from users who don't see the date and then search for hours on why they are not seeing the expected data / results... or worse do mistakes in their packing :(

So their number 1 action is to remove the dates. Which is what we are trying to avoid right? :)

Isn't there a better solution than the dates? Like maybe last OC?

**Comment by kirstenalarsen** [comment]
My vote is for default to remove all dates. If users want to use dates they can fill them in, but agree with @RachL - by far the biggest issue is people misusing the dates and accidentally missing or including some data. Default blank report with no filters, including no dates - that's my vote :

**Comment by Matt-Yorkley** [comment]
> Isn't there a better solution than the dates? Like maybe last OC?

If we ditch our outdated jquery datepickers and replace them with something like this: https://www.daterangepicker.com/
... we could easily add some really nice custom options like "last order cycle" in addition to really nice daterange selection UX.

As things stand on reports performance, I think removing all the dateranges would be suicide for our production servers.

**Comment by RachL** [comment]
> As things stand on reports performance, I think removing all the default dateranges would be suicide for our production servers.

Note that's already what our users are doing. When then land, the first things they do is to remove the dates.
Anyway I agree with you that the date picker must change. Do you know if this one can be easily translated? Another big issue of the current one is that it is English only...

**Comment by kirstenalarsen** [comment]
on the pages where there are no dates the page loads blank to start with. they have to set filters, so it could be server suicide if they actually try to load everything, but that's not the info they want so I really don'w know how much they would do it

**Comment by kirstenalarsen** [comment]
I don't think this should have any of the 'good first issue' or 'hackathon' labels as it currently contains no clear direction / specification of what we want done. Seems like a design question to me - what should we do with it - can I remove the labels to avoid someone picking it up and doing somethign that we don't actually want? @daniellemoorhead @lin-d-hop @RachL

**Comment by RachL** [comment]
I would even close this one. I don't think the solution is the one we want...
